plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -141,6 +145,37 @@ instance Show fun => Pretty (ExBudgetCategory fun) where
 instance ExBudgetBuiltin fun (ExBudgetCategory fun) where
     exBudgetBuiltin = BBuiltinApp
 
+-- | Costs for evaluating AST nodes.  Times should be specified in picoseconds, memory sizes in bytes.
```

Comment:
Perhaps this should go in its own file? It's not very big though.

Reply:
Also, since we have two different things now, maybe `CostModel` should be `BuiltinCostModel`.

Reply:
Did both of these.